api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -31,7 +31,7 @@
  * <p>A ConfigSource will get picked up via the
  * {@link java.util.ServiceLoader} mechanism and must get registered via
  * META-INF/services/javax.config.spi.ConfigSource</p>
- * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
+ * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}. << How?
```

Comment:
So is my understanding correct that `ConfigProvider.getConfig()` will always return the same set of sources once app is booted?
Will `ConfigProvider.getBuilder().withSources(...)` define an alternative config, which will not affect config returned by `ConfigProvider.getConfig()` in any way?